src/use-swr.ts

```diff
@@ -24,7 +24,7 @@ import {
   updaterInterface
 } from './types'
 
-const IS_SERVER = typeof window === 'undefined'
+const IS_SERVER = typeof window === 'undefined' || (typeof Deno !== 'undefined' && !!Deno?.version?.deno)
```

Comment:
is `typeof Deno !== 'undefined'` ok enough for determining deno environment? 
I'm wondering if we can skip the `!!Deno?.versions?.deno` seems it's crashing the tests on CI

Reply:
for safety, better to check the `Deno.version.deno`, i  just updated hope it can pass the tests